src/platforms/unity/migration.mdx

```diff
@@ -10,6 +10,10 @@ redirect_from:
 
 The Sentry SDK deprecated the use of a JSON file to store the options in favor of scriptable objects. If you're migrating from version 0.3.0 or older, make sure to upgrade to version 0.15.0 first, as it is the last version supporting your options stored as JSON. When you're opening the Sentry configuration editor window, that file will be automatically converted into a scriptable object. This support will be dropped in version 0.16.0 and going forward.
 
+## Migrating From 0.7.0 or earlier
```

Comment:
Can you also change the word "earlier" on line 9 to upper case as well.

```suggestion
## Migrating From 0.7.0 or Earlier
```

Reply:
Sure thing!